src/main/java/com/thealgorithms/maths/Perimeter.java

```diff
@@ -1,42 +1,62 @@
 package com.thealgorithms.maths;
 
+// Perimeter of different 2D geometrical shapes
 public class Perimeter {
 
-    public static void main(String[] args) {
-        System.out.println(perimeter_polygon(5, 4));
-        System.out.println(perimeter_rectangle(3, 4));
-        System.out.printf("%,3f", circumference(5));
-    }
-
-    // Perimeter of different 2D geometrical shapes
     /**
-     *Calculate the Perimeter of polygon.
-     * @parameter length of side.
+     * Calculate the Perimeter of polygon
+     * 
      * @parameter number of sides.
+     * @parameter length of side.
      * @return Perimeter of given polygon
      */
-    public static float perimeter_polygon(int n, float side) {
-        float perimeter = n * side;
-        return perimeter;
+    public static float perimeterPolygon(int n, float side) {
```

Comment:
Polygon doesn't have to have equal length sides. It may be different. Make two functions of same name one for equal length sides Polygon, another for variable length (take variable length argument as parameters)

Reply:
> 

ok, then should I also remove perimeterTriangle() and perimeterTrapezoid() and hence replace them with variable-length polygon perimeter calculator function ?

Reply:
Yes, I think, you are right. rather than a triangle, we could have an Equilateral triangle or a polygon that has all the sides of equal length. Just put some comments about what the function is representing